File: VisitBosnia/VisitBosnia.Services/Database/ReviewGallery.cs
using System;
using System.Collections.Generic;

namespace VisitBosnia.Services.Database
{
    public partial class ReviewGallery
    {
        public int Id { get; set; }
        public string? ImageType { get; set; }
        public byte[] Image { get; set; } = null!;
        public int ReviewId { get; set; }

        public virtual Review Review { get; set; } = null!;
    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/reviewGallery.dart/review_gallery.dart
import '../review/review.dart';

class ReviewGallery {
  int? id;
  String? imageType;
  String? image;
  int? reviewId;
  Review? review;

  ReviewGallery(
      {this.id, this.imageType, this.reviewId, this.image, this.review});

  ReviewGallery.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    imageType = json['imageType'];
    reviewId = json['reviewId'];
    image = json['image'];
    // touristFacility = json['touristFacility'];
    review =
        json['review'] != null ? new Review.fromJson(json['review']) : null;
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['imageType'] = this.imageType;
    data['review'] = this.review;
    data['image'] = this.image;
    data['reviewId'] = this.reviewId;
    // data['touristFacility'] = this.touristFacility;
    if (this.review != null) {
      data['review'] = this.review!.toJson();
    }
    return data;
  }
}
